StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/Hint/LinkHintMessageView.swift

```diff
@@ -79,7 +78,12 @@ final class LinkHintMessageView: UIView {
             stackView.centerYAnchor.constraint(equalTo: centerYAnchor),
         ])
         backgroundColor = .linkSurfaceSecondary
-        layer.cornerRadius = Constants.cornerRadius
+
+        if let cornerRadius = LinkUI.appearance.cornerRadius {
+            layer.cornerRadius = cornerRadius
```

Comment:
Not blocking but - why do we have both `LinkUI.appearance.cornerRadius` and `LinkUI.cornerRadius`? When are you supposed to use one vs the other?  Might be some room for improvement here

Reply:
They both resolve to `LinkUI.cornerRadius`, but yes, there's no good answer. Now that we rely on something dynamic, we'll refactor this soon to avoid the static `LinkUI` approach.